assets/src/js/vwo_smart_code.js

```diff
@@ -0,0 +1,23 @@
+/* global vwoSmartScriptCodeData */
+
+if (vwoSmartScriptCodeData?.vwo_account_id) {
+  const vwoScript = `
+    <!-- Start VWO Async SmartCode -->
+    <link rel="preconnect" href="https://dev.visualwebsiteoptimizer.com" />
```

Comment:
I know this is coming from VWO docs but maybe better to use `dns-prefetch` instead. It's a "cheaper" operation and it's consistent with what [we do already](https://github.com/greenpeace/planet4-master-theme/blob/4266c0e68b3dbcb15e027a46ef618581ce19a296/templates/html-header.twig#L40) with other services